src/covid_shared/cli_tools/metadata.py

```diff
@@ -114,7 +114,7 @@ def dump(self, metadata_file_path: Union[str, Path]):
 
 
 def monitor_application(func: types.FunctionType, logger_: Any, with_debugger: bool,
-                        app_metadata: Metadata = Metadata()) -> Callable:
+                        app_metadata: Optional[Metadata] = None) -> Callable:
```

Comment:
tests were failing because of mutable default here, so I changed things slightly